src/netbase.cpp

```diff
@@ -651,7 +667,7 @@ void CNetAddr::SetRaw(Network network, const uint8_t *ip_in)
             memcpy(ip, ip_in, 16);
             break;
         default:
-            assert(!"invalid network");
+	    assert(true);  // invalid network
```

Comment:
Also note that it is not supported to compile Bitcoin or Zcash with NDEBUG: https://github.com/zcash/zcash/blob/zc.v0.11.2.latest/src/main.cpp#L40